README.md

```diff
@@ -24,10 +24,10 @@ PhoneGap plugin for managing application assets with javascript asset maps. Incl
 
 ### downloadFile()
 
-**wizAssets.downloadFile(String URL, String filePathToBeStoredWithFilename, Function success, Function fail);**
+**wizAssets.downloadFile(String remoteURL, String localURI, Function success, Function fail);**
 
 - downloads a file to native App directory @ ./ + gameDir+ / +filePathToBeStoredWithFilename <br />
-- A success returns URI string like; file://documents/settings/img/cards/card001.jpg <br />
+- A success returns a file URI string like; file://documents/settings/img/cards/card001.jpg <br />
```

Comment:
No point anymore, right? I already have the URI. I passed it as the 2nd argument.